fiftyone/utils/sam3.py

```
@@ -0,0 +1,537 @@
+        if self.operation_mode == "concept":
+            item_dict["text_prompts"] = self.text_prompts
+        return item_dict
+
+
+class SegmentAnything3ImageGetItemForVideo(SegmentAnything3ImageGetItem):
+    """Workaround for applying image model to video reader frames.
+
+    Frames are not stored on disk and therefore cannot be loaded.
+    """
+
+    def __call__(self, d):
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__call__`

(ANN204)